### DISCUSSION TITLE
Buttons Menu Bar

### DISCUSSION THREAD

**Discussion by sajkofff** [rejetto/hfs#529] [Ideas]
Hello.  How do I change the ZIP button assignment?  For me it is not relevant.  For example, so that when clicked you can go to another address.![20240324_110129.jpg](https://github.com/rejetto/hfs/assets/48749271/503d473a-02b5-40f2-8f57-3ca1b8ff1d8c)

**Comment by rejetto** [comment]
i don't think this is currently possible, but I'm adding such possibility in 0.53 (unpublished yet, should arrive in a few weeks)

In 0.53 you will be able to do it adding this in the file custom.html
```html
[menuZip]
<a href='http://google.com'><button>Go away</button></a>
```

or using javascript
```js
HFS.onEvent('menuZip', () => `<a href='http://google.com'><button>Go away</button></a>`)
```

or javascript+react
```js
HFS.onEvent('menuZip', () => HFS.h('a', { href: 'http://google.com' }, HFS.h('button', {}, "Go away")))
```

**Reply by sajkofff** [reply]
Thanks

**Comment by djmakers** [comment]
Etot hfs kak ios, nechego ne mogu meneati :D
